persist/sqlite/hosts.go

```diff
@@ -9,12 +9,14 @@ import (
 	"go.sia.tech/explored/explorer"
 )
 
-// HostsForScanning returns hosts ordered by the transaction they were created in.
+// HostsForScanning returns hosts ordered by their time to next scan.  Hosts
+// which are repeatedly offline will face an exponentially growing next scan
+// time to avoid wasting resources.
 // Note that only the PublicKey, V2, NetAddress, and V2NetAddresses fields are
 // populated.
-func (s *Store) HostsForScanning(maxLastScan, minLastAnnouncement time.Time, limit uint64) (result []explorer.Host, err error) {
+func (s *Store) HostsForScanning(now, minLastAnnouncement time.Time, limit uint64) (result []explorer.Host, err error) {
```

Comment:
There's really no benefit to passing the current time in. Go ahead and use `time.Now()` in the SQLite layer